apps/nuxt3-ssr/components/TableRowDetails.vue

```diff
@@ -118,23 +118,25 @@ function buildTOC(sections: ISection[]) {
 }
 
 function sectionTitle(section: ISection) {
-  return section.meta.descriptions
-    ? section.meta.descriptions[0].value
+  let temp = section.meta.labels
+    ? section.meta.labels[0].value
     : section.meta.name;
+  return temp.charAt(0).toUpperCase() + temp.substr(1);
```

Comment:
const + move te reusable and tested function , Case of first letter can be done in css in most cases